backend/fms_core/models/project.py

```diff
@@ -14,28 +14,16 @@
 
```

Comment:
Same. Are those necessary in this PR (different attributes on different lines)? I'd recommend keeping the changes to minimal because we are going to end up with different PRs to merge into the split-sample PR, which will then be merged into master